explorer/graphql/resolvers/test_util.go

```diff
@@ -11,20 +11,76 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 )
 
-func CreateTestBlock(ctx context.Context, t *testing.T, client *ent.Client) {
+func CreateTestBlock(ctx context.Context, t *testing.T, client *ent.Client) ent.Block {
 	t.Helper()
 
-	sourceChainID := uint64(1234)
+	sourceChainID := uint64(1)
 	blockHeight := uint64(0)
 	blockHashBytes := []byte{1, 3, 23, 111, 27, 45, 98, 103, 94, 55, 1, 3, 23, 111, 27, 45, 98, 103, 94, 55}
 	blockHashValue := common.Hash{}
 	blockHashValue.SetBytes(blockHashBytes)
 
-	client.Block.Create().
+	block := client.Block.Create().
 		SetSourceChainID(sourceChainID).
 		SetBlockHeight(blockHeight).
 		SetBlockHash(blockHashValue.Bytes()).
 		SaveX(ctx)
+
+	return *block
+}
+
+// CreateTestBlocks creates n test blocks with n messages and n-1 receipts.
+func CreateTestBlocks(ctx context.Context, t *testing.T, client *ent.Client, count int) {
+	t.Helper()
+	var msg *ent.Msg
+	for i := 0; i < count; i++ {
+		block := CreateTestBlock(ctx, t, client)
+		if msg != nil {
+			createReceipt(ctx, t, client, *msg)
+		}
+		msg = createXMsg(ctx, t, client, block)
+	}
+}
```

Comment:
This is test code (i.e. `CreateTestBlocks` etc) so better put it in `testutil_test.go` with package name `resolvers`.